src/client/src/map-core/components/FitBounder.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { useMap } from 'react-leaflet';
+import { LatLngBounds } from 'leaflet';
+
+export default function FitBounder() {
```

Comment:
nit: Can this name have the word 'Manager' in it? I view it as it manages the bounds of the outer map